lib/material/extra/input_chips/material_input_chips.flow

```diff
@@ -33,12 +39,26 @@ MInputChips(items : Transform<[string]>, style : [MInputChipsStyle]) -> Material
 	acPosScale = make(zeroPositionScale);
 	icPosScale = make(zeroPositionScale);
 	acBaseline = make(0.);
+	
+	button = {
+		if (textButton == "") TEmpty()
+		else MTextButton(
+			textButton, 
+			\-> {
+				dynArrayPush(selectedItems, trim(getValue(text)));
+				nextDistinct(text, "");
+				deferUntilNextFrameRendered(\-> nextDistinct(focused, true))
+			}, 
+			[MPrimaryColor()],
+			[MEnabled(fneq(text, ""))]
+		)
+	}
 
 	autoComplete =
 		MGetManager(\manager -> {
 			defFn = defaultCompletionFn(manager, manager.focus, []).fn;
-			MSelect(items, \itms ->
-				MAutoComplete(text, itms, concatA([
+			MSelect2(items, selectedItems, \itms, selecItms ->
+				MAutoComplete(text, filter(itms,\i -> !contains(selecItms, i)), concatA([
```

Comment:
Don't filter dictionary here, results should be filtered inside MSentenceMatcher.

Reply:
Now I am using MDictionaryDynamic(fselect2